contrib/comdb2makecluster

```diff
@@ -0,0 +1,291 @@
+    fi
+
+    set +e
+    out=
+    # wait until we can query it
+    echo "$DBNAME: waiting until ready"
+    while [[ "$out" != "1" ]]; do
+        out=$(cdb2sql ${CDB2_OPTIONS} --tabs $DBNAME default 'select 1' 2>/dev/null)
+        sleep 1
+    done
+else
+    echo "$DBNAME: copying to cluster"
+    for node in $CLUSTER; do
+        if [ $node == $myhostname ] ; then # skip copying to ourself
+            continue
+        fi
+        copycomdb2 $DBDIR/${DBNAME}.lrl ${node}: &> $DBDIR/logs/${DBNAME}.${node}.copy
+        if [[ $? -ne 0 ]]; then
+            echo "FAILED: copycomdb2 $DBDIR/${DBNAME}.lrl ${node}: "
+            echo "see $DBDIR/logs/${DBNAME}.${node}.copy "
+            exit 1
+        fi
+    done
+
+    echo "export COMDB2_ROOT=$DBDIR" >> ${DBDIR}/replicant_vars
+    echo "export PATH=$PATH" >> ${DBDIR}/replicant_vars
+    CMD="source ${DBDIR}/replicant_vars ; $comdb2task ${DBNAME} -lrl $DBDIR/${DBNAME}.lrl"
+    echo "$DBNAME: starting"
+    for node in $CLUSTER; do
+        if [ $node == $myhostname ] ; then # dont ssh to ourself -- just start db locally
+            if [[ -n ${DEBUG_PREFIX} && ${INTERACTIVE_DEBUG} -eq 1 ]]; then
+                echo -e "$DBNAME: Execute the following command on ${node}: ${TEXTCOLOR}${DEBUG_PREFIX} $comdb2task ${DBNAME} -lrl $DBDIR/${DBNAME}.lrl -pidfile ${TMPDIR}/${DBNAME}.${node}.pid${NOCOLOR}"
+            else
+                ${DEBUG_PREFIX} $comdb2task ${DBNAME} -lrl $DBDIR/${DBNAME}.lrl -pidfile ${TMPDIR}/${DBNAME}.${node}.pid 2>&1 | gawk '{ print strftime("%H:%M:%S>"), $0; fflush(); }' >$DBDIR/logs/${DBNAME}.${node}.db 2>&1 &
+            fi
+            continue
+        fi
+
+        if [[ -n ${DEBUG_PREFIX} && ${INTERACTIVE_DEBUG} -eq 1 ]]; then
+            echo -e "$DBNAME: Execute the following command on ${node}: ${TEXTCOLOR}${DEBUG_PREFIX} ${CMD}${NOCOLOR}"
+        else
+            scp -o StrictHostKeyChecking=no ${DBDIR}/replicant_vars $node:${DBDIR}/replicant_vars &>> $DBDIR/setup.log
+            # redirect output from CMD to a subshell which runs awk to prepend time
+            ssh -o StrictHostKeyChecking=no -tt $node ${DEBUG_PREFIX} ${CMD} 2>&1 </dev/null > >(gawk '{ print strftime("%H:%M:%S>"), $0; fflush(); }' > $DBDIR/logs/${DBNAME}.${node}.db)  &
+            # $! will be pid of ssh (if we had used pipe, $! would be pid of awk)
+            echo $! > ${TMPDIR}/${DBNAME}.${node}.pid
+        fi
+    done
+
+    set +e
+    echo "$DBNAME: waiting until ready"
+    sleep 1
+    for node in $CLUSTER; do
+        out=$(cdb2sql ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' 2>&1)
+        while  [[ "$out" != "1" ]]; do
+            sleep 2
+            out=$(cdb2sql ${CDB2_OPTIONS} --tabs --host $node $DBNAME 'select 1' 2>&1)
+        done
+    done
+    n=$(cdb2sql ${CDB2_OPTIONS} -tabs $DBNAME --host $node 'exec procedure sys.cmd.send("bdb cluster")' | grep "lsn" | wc -l)
```

Comment:
You generate a config file, mention that it can be used to configure the destination?  Up to you.